### ISSUE TITLE
EfficientNet pretrained on Open Images

### ISSUE THREAD

**Issue by harsha070** [tensorflow/tpu#440]
When can I expect pre-trained checkpoints of EfficientNet on open images dataset to be released? If already available, kindly leave the link below.

Thanks.

**Comment by saberkun** [comment]
That's a good idea. We do not have it yet.
Will share if there is any update.
@mingxingtan

**Comment by harsha070** [comment]
Sure. Thanks.

**Comment by harsha070** [comment]
Is there an update?

**Comment by mingxingtan** [comment]
I have not trained EfficientNets on OpenImage before.  Is there any specific reason that you need OpenImage-pretrained checkpoints?

**Comment by harsha070** [comment]
Well, the best performance on OpenImages (Classification Task) is by Resnet 101 [link](https://storage.googleapis.com/openimages/web/extras.html) trained on OpenImages v2 (2017) which is quite old. I wanted to see how EfficientNets compare with ResNet 101. Sadly, the dataset is very large for me to train and hence asking for help!

**Comment by SimJeg** [comment]
@mingxingtan thank you for this impressive work ! I work on medical images and pretrained models are keys because we face to small datasets. Thus the better a model transfer the better for us. Do you think pretraining efficient net on larger datasets (open images or jft for instance) would even improve the transferability of the model ?

**Comment by VoldemortGin** [comment]
Any update? I am training EfficientNet-b7 on a medical image dataset. But unlike densenet or resnet, it seems that EfficientNet takes much more time to train (I mean b7. b3 is very fast but enough accuracy for me). I think if there is an pretrained on specific  dataset would be much more convenient for everyone.

**Comment by yutkin** [comment]
+, OpenImages is very desirable not only because it has many more classes compared to ImageNet, but it is a multilabel task.
